### ISSUE TITLE
[startups]: organisations page has several accessibility issues for screen readers users

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#1818]
Our [startups organisation](https://www.gitpod.io/for/startups/organisations) page has several accessibility issues for screen reader users i.e the Search input is not labelled correctly + the side navigation has no usability context whatsoever.

**Comment by mikenikles** [comment]
I see an a11y score of 100 in Lighthouse. Please re-open if this issue is still relevant.

**Comment by nisarhassan12** [comment]
Thanks, @mikenikles the Lighthouse audits can't detect the accessibility issues mentioned in the above comment.

**Comment by mikenikles** [comment]
👍 Thanks for the additional insights. Do you know if there's a tool that detects these screen reader issues so that we could link to it and see what needs to be addressed?

**Comment by nisarhassan12** [comment]
I detected these issues by manually browsing the page via Voiceover on my Mac OS.
